src/utils/fetchClient.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Try to avoid using `eslint-disable` for specific rules. If it's absolutely necessary, add a comment explaining why you had to disable the rule. 📝